gamestates/cSetupSkirmishGameState.cpp

```diff
@@ -640,10 +635,7 @@ void cSetupSkirmishGameState::prepareSkirmishGameToPlayAndTransitionToCombatStat
         int maxDistance = worms * 32; // 128 / 4
         int wormCell = map.getRandomCell();
         int failures = 0;
-        char msg[255];
-        sprintf(msg, "Skirmish game with %d sandworms, minDistance %d, maxDistance %d", worms, minDistance,
-                maxDistance);
-        logbook(msg);
+        logbook(fmt::format("Skirmish game with {} sandworms, minDistance {}, maxDistance {}", worms, minDistance, maxDistance));
```

Comment:
All the fmt conversion is appreciated though! I wish it was a separate commit. Then we could have merged it already.